src/mozanalysis/experiment.py

```diff
@@ -1199,12 +1197,6 @@ def for_ts(
     @first_enrollment_date.validator
     def _validate_first_enrollment_date(self, attribute, value):
         assert self.first_enrollment_date <= self.last_enrollment_date
-        assert self.first_enrollment_date <= self.first_date_data_required
-        assert self.first_enrollment_date <= self.last_date_data_required
-
-    @last_enrollment_date.validator
-    def _validate_last_enrollment_date(self, attribute, value):
-        assert self.last_enrollment_date <= self.last_date_data_required
```

Comment:
It seems like these should still be applicable, no?

Reply:
With pre-enrollment analyses, the `last_date_data_required` is the day before the `first_enrollment_date`